src/formatDate.js

```diff
@@ -8,7 +8,23 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[fromFormat.length - 1];
+  const newData = date.split(separator);
+  const newFormat = Array.from(toFormat);
```

Comment:
Creating a copy of the `toFormat` array using `Array.from()` is unnecessary because you do not modify the original array. You can directly use `toFormat` and pop the last element which is the new separator.